src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -100,6 +100,11 @@ public PaxTarEntry(TarEntryType entryType, string entryName, IEnumerable<KeyValu
         /// <summary>
         /// Initializes a new <see cref="PaxTarEntry"/> instance by converting the specified <paramref name="other"/> entry into the PAX format.
         /// </summary>
+        /// <param name="other">The <see cref="TarEntry"/> instance to convert to the PAX format.</param>
+        /// <exception cref="InvalidOperationException"><para><paramref name="other"/> is a <see cref="PaxGlobalExtendedAttributesTarEntry"/> instance.</para>
+        /// <para>-or-</para>
+        /// <para>The entry type of <paramref name="other"/> is not supported in PAX.</para></exception>
```

Comment:
For consistency with the other suggestions.
```suggestion
        /// <para>The entry type of <paramref name="other"/> is not supported in the PAX format.</para></exception>
```